includes/class-newspack-popups-model.php

```diff
@@ -859,39 +887,48 @@ public static function container_style( $popup ) {
 	}
 
 	/**
-	 * Endpoint to dismiss Pop-up.
+	 * Endpoint to report Pop-up data.
 	 *
 	 * @return string Endpoint URL.
 	 */
-	public static function get_dismiss_endpoint() {
-		return str_replace( 'http://', '//', get_rest_url( null, 'newspack-popups/v1/reader' ) );
+	public static function get_reader_endpoint() {
+		return str_replace(
+			'http://',
+			'//',
+			plugins_url( '../api/', __FILE__ )
+		);
 	}
 
 	/**
-	 * Generate hidden fields to be used in all dismiss FORMs.
+	 * Generate hidden fields to be used in all forms.
 	 *
 	 * @param  object $popup A Pop-up object.
 	 * @return string Hidden fields markup.
 	 */
 	public static function get_hidden_fields( $popup ) {
 		ob_start();
 		?>
-		<input
-			name="url"
-			type="hidden"
-			value="CANONICAL_URL"
-			data-amp-replace="CANONICAL_URL"
-		/>
 		<input
 			name="popup_id"
 			type="hidden"
-			value="<?php echo ( esc_attr( $popup['id'] ) ); ?>"
+			value="<?php echo self::canonize_popup_id( esc_attr( $popup['id'] ) ); ?>"
 		/>
+		<input
```

Comment:
Nit: the whitespace is weird in just this `input`.

Reply:
Done in 856d73618a30aca8f7940edb5b4983225d675b97